spec/requests/api/v2/access_request_spec.rb

```diff
@@ -282,12 +282,12 @@
       it { should have_http_status(:unauthorized) }
     end
 
-    context 'when unauthorized' do
-      let(:unauthorised_user) { create(:user) }
-      let(:payload) { { email: unauthorised_user.email } }
+    context 'allows non admin users to be authorized' do
+      let(:non_admin_user) { create(:user) }
+      let(:payload) { { email: non_admin_user.email } }
 
-      it "should raise an error" do
-        expect { do_post }.to raise_error Pundit::NotAuthorizedError
+      it "should not raise an error" do
+        expect { do_post }.not_to raise_error
```

Comment:
Feel like this test is a bit redundant? Everything should *generally* not error. 

Does it make more sense for this to return a `200` response or something more explicit that the expected behaviour has worked.

Reply:
Have done 👍